n64.mk

```diff
@@ -1,5 +1,6 @@
 BUILD_DIR ?= .
 SOURCE_DIR ?= .
+FILE_PREFIX ?=
```

Comment:
I'd rather to move this down together with the other N64 macros, and with a comment